packages/ui/src/components/primitives/Message.stories.tsx

```diff
@@ -0,0 +1,108 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { Message } from './Message';
+import {
+	mockDiscordAccount,
+	mockMessageWithDiscordAccount,
+} from '~ui/test/props';
+
+const meta = {
+	component: Message,
+	parameters: {
+		a11y: {
+			disable: true,
+		},
+	},
+} as Meta<typeof Message>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Primary: Story = {
+	args: { message: mockMessageWithDiscordAccount() },
+};
+
+export const OverflowLetters: Story = {
+	args: {
+		...Primary.args,
+		message: {
+			...mockMessageWithDiscordAccount(),
+			content:
+				'Helloooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo',
```

Comment:
this currently doesnt wrap on overflow, plz fix

Reply:
Fixed 👍